web/src/views/CalendarLinkGeneratorView.vue

```diff
@@ -179,8 +283,8 @@ export default defineComponent({
                 calendarForm.value.startTime
             )}%2F${convertDateTimeToPureISO(calendarForm.value.endTime)}&ctz=${
```

Comment:
variables should have spaces at the start and end like this => `${ variableName }`